src/Calypso-SystemPlugins-Reflectivity-Browser/ClyRemoveWatchpointCommand.class.st

```diff
@@ -19,7 +19,7 @@ ClyRemoveWatchpointCommand class >> contextMenuOrder [
 ]
 
 { #category : #execution }
-ClyRemoveWatchpointCommand >> metalinkManagerClass [
+ClyRemoveWatchpointCommand >> handlerClass [
```

Comment:
What this handler is about? What handlerClass handles?
Original name was about idea that these kind of metalinks are managed by system: system maintains registries of breakpoints, watchpoints and counters. I thought that they could be refactored to have common management code.

Reply:
I just think about better name because now we have quite strange abstract method ClyDebuggingCommand>>handlerClass

Reply:
To me they are not metalinks, they are tools using metalinks.
For instance you always call the `Breakpoint` api and not methods from the metalink it holds.
So you it's not managing metalinks. Tools are using them, but then you do not need to know that in the commands, only that you delegate work to those "tools" that "handle" your command action.
To me `handleClass`was more appropriate.
We can find a better name.

Reply:
I agree with you. Maybe #debuggingToolClass?
And to solve code duplication in those tools we could introduce DebuggingTool as a common superclass.

Reply:
Yes that's good.
I updated.